docs/bigmaven-testing.md

```diff
@@ -0,0 +1,21 @@
+# Testing MDS changes in Bigmaven before release
```

Comment:
```suggestion
# Testing Changes in Bigmaven 
```

Reply:
Actually, we can probably just generalize this:

```suggestion
# Testing Unreleased Changes
```

The fact that we use bigmaven is just because it's our biggest use-case. But I can see that in the future we just want to make sure all our use-cases are tested.